daemon/container_unix.go

```diff
@@ -37,8 +37,11 @@ import (
 	"github.com/opencontainers/runc/libcontainer/label"
 )
 
+// DefaultPathEnv is unix style list of directories to search for executables.
```

Comment:
I'd suggest clarifying this by explicitly saying that the list is separated by : characters.